tests/bdd/zfs.feature

```diff
@@ -139,4 +139,133 @@ Feature: Validate volume provisioning for fsType zfs, ext4, xfs, btrfs
       |  ZVOL       | 8192          | 16384      |
       |  ZVOL       | 8192          | 16384      |
 
-    
\ No newline at end of file
+    
+############################################################################################################
+################## The below bdd need to be implemented in ci tests ########################################
+############################################################################################################
+
+
+  Scenario Outline: Online volume resize
+    Given a storage class is created with fsType as <fsType>
+    And pvc is created referencing this same storage class with zfsvolume in Ready state
+    And a deployment is created using the same pvc which is in running state
+    When resize volume requested is by updating the PVC yaml
+    Then corresponding pvc must be resized
+    Examples:
+      | fsType |
+      |  zfs   |
+      |  ext4  |
+      |  xfs   |
+
+  Scenario Outline: Create a snapshot from a pvc for raw block volume
+    Given a storage class is created without any fstype
+    And a pvc with volumeMode as Bolck is created referencing this storage class
+    And a deployment is created using the same pvc
+    When the snapshot create command for the given pvc is run in the same namespace
+    Then the snapshot must be created for the corresponding pvc with status as ready to use
```

Comment:
Need step to validate the dataset.

Reply:
what do you mean by this, its a raw block